ui/pages/onboarding-flow/recovery-phrase/recovery-phrase-chips.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import Chip from '../../../components/ui/chip';
+import Box from '../../../components/ui/box';
+import Typography from '../../../components/ui/typography';
+import { ChipWithInput } from '../../../components/ui/chip/chip-with-input';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import {
+  TYPOGRAPHY,
+  COLORS,
+  BORDER_STYLE,
+  SIZES,
+  DISPLAY,
+} from '../../../helpers/constants/design-system';
+
+export default function RecoveryPhraseChips({
+  seedPhrase,
+  seedPhraseRevealed,
+  confirmPhase,
+  setInputValue,
+  inputValue,
+  indicesToCheck,
+}) {
+  const t = useI18nContext();
+  const hideSeedPhrase =
```

Comment:
Are we specifically looking to see if `seedPhraseRevealed === false`, or `false || null`?

Reply:
Yeah that's a nicer way to put it.

Reply:
Done!